src/Components/Users/UserAdd.tsx

```diff
@@ -326,7 +327,7 @@ export const UserAdd = (props: UserProps) => {
 
   const setFacility = (selected: FacilityModel | FacilityModel[] | null) => {
     setSelectedFacility(selected as FacilityModel[]);
-    const form = { ...state.form };
+    const form: any = { ...state.form };
```

Comment:
Casting to `any` not allowed.

```suggestion
    const form = { ...state.form };
```